MAINTAINERS.md

```diff
@@ -33,12 +35,16 @@ We enthusiastically welcome contributors to create addons, whether within or out
         * Jian Qiu
         * David Eads
     * `registration`
-        * Amrutha Harikumar
         * Gaurav Jaswal
         * Le Yang
-        * Suvaansh Kumar
         * Wei Liu
         * Zhao Xue
+    * `awsirsa registration`
```

Comment:
@qiujian16 could you approve this division change? Thanks.

@jaswalkiranavtar could you DM me Jeffrey and Ramesh's email address? Thanks.

Reply:
Shared emails in DM.